Release: slate-cbl v2.1.17

nbey opened the pull request:
- [x] Implement course section sorting properly in accordance to [Slate v2.1.22](https://github.com/SlateFoundation/slate/pull/99)
- [x] Remove SectionsRequestHandler override
- [x] Update README and setup instructions
- [x] Fix content area report issues

nbey committed:
Implement course section sorting properly

themightychris commented:
@nbey excellent work! Just delete the old query param in the slate PR and you can go ahead and merge both

BillClinton committed:
Fix content area report issues

These changes were originally committed to the wrong branch (issues/rate-tasks) and
I am now committing them to develop.

BillClinton committed:
Remove merge conflict note

The code highlighted by the merge conflict syntax was meant to be there.

themightychris committed:
Update release branch name

Commit:
Remove SectionsRequestHandler override from git mappings

themightychris committed:
Update README.md

themightychris committed:
Merge pull request #372 from SlateFoundation/issues/setup-guide

Update setup process
